components/events/summer-school/FaqSection.vue

```diff
@@ -0,0 +1,323 @@
+<template>
+  <section class="faq-section">
+    <h2 class="copy__title" v-text="'Frequently asked questions'" />
```

Comment:
Please use `v-text` when binding to a variable. When using a string literal, we usually go with:
```suggestion
    <h2 class="copy__title">
      Frequently asked questions
    </h2>
```